datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/manifest.py

```diff
@@ -65,22 +68,47 @@ def is_metric_in_metadata_file(metric, check):
     return False
 
 
+def get_original_manifest_field(check_name, field_to_check, repo_url, manifests_from_master):
+    repo_url = f'{repo_url}/{check_name}/manifest.json'
+    if manifests_from_master.get(check_name):
+        data = manifests_from_master.get(check_name)
+    else:
+        data = requests.get(repo_url.format(check_name))
+        manifests_from_master[check_name] = data
+
+    if data.status_code == 404:
+        # This integration isn't on the provided repo yet, so it's field can change
+        return None
+
+    data.raise_for_status()
+    decoded_master = data.json()
+    return decoded_master.get(field_to_check)
+
+
 @click.command(context_settings=CONTEXT_SETTINGS, short_help='Validate `manifest.json` files')
 @click.option('--fix', is_flag=True, help='Attempt to fix errors')
-@click.option('--include-extras', '-i', is_flag=True, help='Include optional fields')
+@click.option('--include-extras', '-i', is_flag=True, help='Include optional fields, and extra validations')
+@click.option(
+    '--repo_url', '-r', help='The raw github URL to the base of the repository to compare manifest fields against'
+)
 @click.pass_context
-def manifest(ctx, fix, include_extras):
+def manifest(ctx, fix, include_extras, repo_url):
     """Validate `manifest.json` files."""
     all_guids = {}
+    raw_gh_url = 'https://raw.githubusercontent.com/DataDog/{}/master/'
 
     root = get_root()
     is_extras = ctx.obj['repo_choice'] == 'extras'
-
+    formatted_repo_url = repo_url or raw_gh_url.format(os.path.basename(root))
     ok_checks = 0
     failed_checks = 0
     fixed_checks = 0
+    manifests_from_master = {}
+
     echo_info("Validating all manifest.json files...")
     for check_name in sorted(os.listdir(root)):
+        if check_name != 'new_integration':
+            continue
```

Comment:
Looks like an oversight, but these two lines make it so that no manifest validation is run at all anymore since this PR was merged, unless for the `new_integration` folder! Oops. 😄 Pushing a PR to remove these to merge after freeze…

Reply:
Oh man. Really sorry about this :facepalm: This was indeed a debug line that didn't get removed :(